### PR TITLE
Added details on usage, modified main to accept arguments

### PR DESCRIPTION
**Pull request by archetana**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
 # huffman-image-compression
-Huffman Image Compression
+Huffman Image Compression is one of the basic compression methods, that have proven useful in image and video compression standards. When applying Huffman encoding technique on an Image, the source symbols can be either pixel intensities of the Image, or the output of an intensity mapping function.
+
+# Usage
```
Comment (nikramakrishnan) [inline README.md:4]: This looks like it should be within three backticks ```

**Review by nikramakrishnan — CHANGES_REQUESTED** [review]
Great work! Just a small change required.

File: README.md (hunk 1 above)
Comment (archetana) [reply README.md:4]: Fixed

**Review by nikramakrishnan — APPROVED** [review]

**Comment by nikramakrishnan** [comment]
Thanks much! 😄 👏  Merging this. Happy hacktober!

**Pull request merged**